README.md

```diff
@@ -67,6 +67,14 @@ For example:
     // Other callbacks
   }));  
 ```
+### `preRender`
+
+Callback function called before rendering `App`. This callback should return a valid component.
```

Comment:
Just need to make it clear here that this component can be resolved in a `Promise`